src/Forms/EditFormFactory.php

```diff
@@ -34,10 +35,17 @@ protected function getFormFields(RequestHandler $controller = null, $name, $cont
     {
         $fields = parent::getFormFields($controller, $name, $context);
 
-        /** @var HTMLEditorField $contentField */
-        $contentField = $fields->fieldByName('Root.Main.HTML');
-        if ($contentField) {
-            $contentField->setRows(5);
+        // Configure a slimmed down HTML editor for use with blocks
+        /** @var HTMLEditorField $editorField */
+        $editorField = $fields->fieldByName('Root.Main.HTML');
+        $editorField->setRows(7);
```

Comment:
I think you need to put the if statement back - this form factory applies to all blocks, not just content blocks - you may get a null pointer exception

Reply:
I've updated this